CLI errors if you just type `dnote edit`

e.g.

```
▶ dnote edit
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/dnote/dnote/cli/cmd/edit.newRun.func1(0xc0000cef00, 0x10766e8, 0x0, 0x0, 0x0, 0x0)
	/ext-go/1/src/github.com/dnote/dnote/cli/cmd/edit/edit.go:79 +0xb0a
github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra.(*Command).execute(0xc0000cef00, 0x10766e8, 0x0, 0x0, 0xc0000cef00, 0x10766e8)
	/ext-go/1/src/github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1024fe0, 0xc000134780, 0xc0000e7cf0, 0x81c6fc)
	/ext-go/1/src/github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra.(*Command).Execute(0x1024fe0, 0x1, 0xc000134780)
	/ext-go/1/src/github.com/dnote/dnote/cli/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/dnote/dnote/cli/cmd/root.Execute(0xc000134780, 0xa)
	/ext-go/1/src/github.com/dnote/dnote/cli/cmd/root/root.go:43 +0x2d
main.main()
	/ext-go/1/src/github.com/dnote/dnote/cli/main.go:76 +0x58c
```

--- a/cli/cmd/edit/edit.go
+++ b/cli/cmd/edit/edit.go
@@ -58,7 +58,7 @@ func NewCmd(ctx infra.DnoteCtx) *cobra.Command {
 }
 
 func preRun(cmd *cobra.Command, args []string) error {
-	if len(args) > 2 {
+	if len(args) != 1 && len(args) != 2 {
 		return errors.New("Incorrect number of argument")
 	}
 

